src/autofit/afglobal.c

```diff
@@ -357,7 +357,7 @@
 
 #ifdef FT_CONFIG_OPTION_USE_HARFBUZZ
     globals->hb_font = ft_hb_ft_font_create( face, NULL );
-    globals->hb_buf  = hb_buffer_create();
+    globals->hb_buf  = hb(buffer_create)();
```

Comment:
I find this to be rather ugly/confusing. Can’t we define `hb_buffer_create` etc in `ft-hb.h` using the `FT_HB_APIS` macro?

Reply:
That certainly make the code cleaner. But I didn't like the idea of stepping on HB symbols. Sure. I'll go that direction. Easier for me.

Reply:
> I find this to be rather ugly/confusing. Can’t we define `hb_buffer_create` etc in `ft-hb.h` using the `FT_HB_APIS` macro?

Oh no, we can't. You can't define a macro using a macro...

Reply:
So we'd need another list of:
```
#define hb_buffer_create ft_hb_buffer_create
```
Sure, doable.

Right now I have all the used symbols in one place only.

Reply:
Right, that settles it then

Reply:
One can argue either way (keeping a single list of symbols vs prettier code), so I’d leave it that way unless someone else feels strongly about it.

Reply:
I'll leave it to Werner.

Reply:
Another option is to use `ft_hb(hb_buffer_destroy)` instead of `hb(buffer_destroy)`.